tests/e2e/webapp.spec.ts

```diff
@@ -384,6 +384,48 @@ test.describe('developer settings', () => {
     await expect(page.getByLabel('Retrieval mode')).toContainText('Vectors + Text (Hybrid)');
   });
 
+  test('enable branding toggled', async ({ page }) => {
+    await page.goto('/');
+    await page.getByTestId('button__developer-settings').click();
+    // toggle enable branding
+    await page.locator('label').filter({ hasText: 'Enable branding' }).click();
+    await page.waitForTimeout(1000);
+    // await for brading to be visible
+    await expect(page.getByTestId('chat-branding')).toBeVisible();
+  });
+
+  test('select dark theme', async ({ page }) => {
+    await page.goto('/');
+    await page.getByTestId('button__developer-settings').click();
+    await page.locator('label').filter({ hasText: 'Select theme' }).click();
+    await page.waitForTimeout(1000);
```

Comment:
generally timeout is prone to flakiness. It's better to wait for an element state instead. You can increase the timeout on the element check instead. (same for all the waitTimeouts below)